src/vmm/tests/integration_tests.rs

```diff
@@ -369,7 +330,7 @@ fn verify_load_snapshot(snapshot_file: TempFile, memory_file: TempFile) {
                 microvm_state,
                 mem,
                 false,
-                &empty_seccomp_filter,
+                &mut empty_seccomp_filters,
```

Comment:
Wouldn't restoring the uVM from snapshot reinstall the same seccomp filters?
Why do we need to pass an empty seccomp filters here?